db/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:16
+ENV POSTGRES_USER=postgres
+ENV POSTGRES_PASSWORD=1234
```

Comment:
What do you think to move pass to env variable and do not store in version control system?

Reply:
Great advise, it have been fixed by now in the 0.04 commit. Passed the username as well.